test/e2e/database_test.go

```diff
@@ -0,0 +1,267 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"os/exec"
+	"strconv"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	olsv1alpha1 "github.com/openshift/lightspeed-operator/api/v1alpha1"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	StorageClassNameLocal  = "local-storage"
+	StorageClassNameCI     = "gp3-csi"
```

Comment:
Can't we have a function to get the name of the storageClass used by default in the cluster?
Because, for example, 4.16 uses gp3-csi. But 4.17 uses standard-csi. It got renamed. Which means this test will fail in 4.17 OCP clusters.
Don't know how it is in hypershift tho